src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
+    m_pInputLatencyCompensationDelay = new EngineDelay(group,
+        ConfigKey(group, "inputLatency"));
+    m_pInputLatencyCompensationHeadphonesDelay = new EngineDelay(group,
+        ConfigKey(group, "headMixDelay"));
```

Comment:
If the user does not wish to hear their voice with a delay, they can either listen to the booth output excluding the talkover mix or use direct monitoring. I suspect that the real reason users have complained about hearing their voice in the master output delayed is not because it is delayed, but because it has been mixed out of time without latency compensation and users have not understood what the problem was.